packages/fuel-gauge/src/predicate/predicate-duplication.test.ts

```diff
@@ -0,0 +1,356 @@
+import { WalletUnlocked } from '@fuel-ts/account';
+import { launchTestNode } from 'fuels/test-utils';
+
+import { PredicateWithConfigurable } from '../../test/typegen';
+import type {
+  PredicateWithConfigurableConfigurables,
+  PredicateWithConfigurableInputs,
+} from '../../test/typegen/predicates/PredicateWithConfigurable';
+
+import { fundAccount, assertBalance } from './utils/predicate';
+
+/**
+ * @group node
+ * @group browser
+ */
+describe('Predicate.fromInstance', () => {
+  const amountToPredicate = 300_000;
+
+  const defaultValues = {
+    FEE: 10,
+    ADDRESS: '0x38966262edb5997574be45f94c665aedb41a1663f5b0528e765f355086eebf96',
+  };
+
+  describe('Basic Instance Operations', () => {
```

Comment:
Anything in `fuel-gauge` is inherently end to end as we are testing a predicate against a live provider. 

These `Basic Instance Operations` tests _could_ be moved to the `test` directory inside the account package. Or I would prefer if we got rid of the nested `describe` here and on line `184` detailing the e2e groupings. These sub groupings are redundant IMO.